core/src/js/components/notifications.component.ts

```diff
@@ -77,7 +80,28 @@ export class NotificationsComponent implements AfterViewInit, OnDestroy {
 		private ow: OverwolfService,
 		private elRef: ElementRef,
 		private prefs: PreferencesService,
-	) {}
+	) {
+		this.notifications$ = this.ow.getMainWindow().notificationsEmitterBus;
+		this.windowId$ = from(this.ow.getCurrentWindow()).pipe(pluck('id'));
+
+		this.notifications$.pipe(
+			delayWhen(() => this.windowId$),
+			withLatestFrom(this.windowId$),
+			filter(([message, windowId]) => !!message),
```

Comment:
You asked why I used `!!` and it got me thinking. `filter` needs to return a boolean which message is not.

We could do `filter(message => { if(message) { return true; } else { return false; } })`
Which could be shortened to `filter(message => message ? true : false)`
Or even shorter `filter(message => !!message)`

Not that shorter is better, but that explains how I got in the habit of using `!!` with `filter`. Just wanted to follow up, cheers

Reply:
Got it. When using `filter()` on arrays, it is enough to just do `filter(message => message)`, which removes null-ish items. I guess the signature is different then, and only accept real booleans, not true-ish values?

Reply:
Yeah, what's weird is that the build will mention an error, but it still seems to work anyway with just `filter(message => message)`